### PR TITLE
feat: Add some tweaks to the overall style of the components

### PR DESCRIPTION
**Pull request by GuillaumeAmat**
@iOiurson I made some tweaks, nothing fancy, juste to be more homogeneous.

### CODE REVIEW

File: src/components/Notification/Notification.jsx [hunk 1]
```diff
@@ -127,27 +127,22 @@ const StyledAside = styled.aside`
...
-
   .close-btn {
     display: inline-block;
-    color: ${props => props.theme.controlColor};
-    background: transparent;
-    border-width: 0;
-    height: 100%;
-    padding: 0 1rem;
-    margin-left: 0.5rem;
+    position: absolute;
```
Comment (bleucitron) [inline src/components/Notification/Notification.jsx:133]: Could you verify the positioning of the cross ? It's not always correctly centered.

**Review by bleucitron — CHANGES_REQUESTED** [review]

File: src/themes/Default.jsx [hunk 2]
```diff
@@ -270,39 +271,43 @@ export const config = {
...
       color: colors.darkGray2,
+      disabledOpacity: 0.6,
+
       sm: {
         height: '2.2rem',
         size: '1.4rem',
         top: '0.7rem',
         innerSize: '1.1rem',
-        paddingLeft: '2rem'
+        paddingLeft: '1rem'
```
Comment (bleucitron) [inline src/themes/Default.jsx]: These padding values don't seem right anymore, the checkbox is overlapping with the label.

File: src/themes/Default.jsx [hunk 3]
```diff
@@ -270,39 +271,43 @@ export const config = {
...
+
       sm: {
         height: '2.2rem',
         size: '1.4rem',
         top: '0.7rem',
         innerSize: '0.8rem',
         innerTop: '1rem',
         innerLeft: '0.3rem',
-        paddingLeft: '2rem'
+        paddingLeft: '1rem'
```
Comment (bleucitron) [inline src/themes/Default.jsx]: These padding values don't seem right anymore, the checkbox is overlapping with the label.

**Review by bleucitron — CHANGES_REQUESTED** [review]

File: src/components/Notification/Notification.jsx (hunk 1 above)
Comment (GuillaumeAmat) [reply src/components/Notification/Notification.jsx:133]: It is fixed!

**Comment by GuillaumeAmat** [comment]
@iOiurson It is good to go!

**Comment by GuillaumeAmat** [comment]
Crap, the coverage check is failing because of the merge from `develop`... @iOiurson You can't merge with your admin rights or add some tests on the new components you added. As you want

**Comment by bleucitron** [comment]
Hmmm I don't know, some positioning still feel weird on my Chrome...
Do you mind if we give a final touch on this together IRL, so we both have our eyes available to each other ?

About the coverage, I was thinking we can forget about it for now, and have an issue to complete missing tests.

**Comment by GuillaumeAmat** [comment]
> Hmmm I don't know, some positioning still feel weird on my Chrome...

Crap...

> Do you mind if we give a final touch on this together IRL, so we both have our eyes available to each other ?

Sure!

> About the coverage, I was thinking we can forget about it for now, and have an issue to complete missing tests.

No problem for me.

**Comment by GuillaumeAmat** [comment]
@iOiurson Do you agree to merge that PR and open specific issues about your "weird Chrome behaviors"?

Or maybe giving me the list of that behaviors?

Thanks!

**Comment by bleucitron** [comment]
Ok, merging.

**Comment by GuillaumeAmat** [comment]
👌

**Pull request merged**